LinkageError when using camel-quarkus-activemq together with quarkus-agroal

Java: 11.0.2 (Oracle Corporation 11.0.2+9)
Gradle: 7.3.3
Quarkus: 2.7.1.Final

When using `camel-quarkus-activemq` together with `quarkus-agroal` there is a LinkageError:

```
Caused by: java.lang.LinkageError: loader constraint violation in interface itable initialization for class io.agroal.pool.util.XAConnectionAdaptor: when selecting method 'javax.transaction.xa.XAResource javax.sql.XAConnection.getXAResource()' the class loader 'platform' for super interface javax.sql.XAConnection, and the class loader io.quarkus.bootstrap.classloading.QuarkusClassLoader @2e008502 of the selected method's class, io.agroal.pool.util.XAConnectionAdaptor have different Class objects for the type javax.transaction.xa.XAResource used in the signature (javax.sql.XAConnection is in module java.sql of loader 'platform'; io.agroal.pool.util.XAConnectionAdaptor is in unnamed module of loader io.quarkus.bootstrap.classloading.QuarkusClassLoader @2e008502, parent loader 'app')
```

In order to get this working you have to exclude the transitive `geronimo-jta_1.1_spec`  dependency included by `camel-quarkus-activemq`, e.g.:

```groovy
implementation ('org.apache.camel.quarkus:camel-quarkus-activemq') {
  exclude group: 'org.apache.geronimo.specs', module: 'geronimo-jta_1.1_spec'
}
```

Expected is that there is no linkage error and that the dependencies work together without any extra configuration.

--- a/poms/bom/pom.xml
+++ b/poms/bom/pom.xml
@@ -123,6 +123,10 @@
                         <groupId>commons-logging</groupId>
                         <artifactId>commons-logging</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.geronimo.specs</groupId>
+                        <artifactId>geronimo-jta_1.1_spec</artifactId>
+                    </exclusion>
                 </exclusions>
             </dependency>
             <dependency>

